### DISCUSSION TITLE
How to clean different dirty pages using the same command

### DISCUSSION THREAD

**Discussion by srini8080** [ImageMagick/ImageMagick#5192] [Digital Image Processing]
I have dirty pages of various severity. Can I get one single command to clean them all.

-threshold 40% -deskew 20% is working for 2 of them and
-threshold 60% -deskew 10% is working for 1 of them

Enclosed are the 3 images.

Thanks,

![00035](https://user-images.githubusercontent.com/12046302/172040057-acc717da-037b-4202-a4d8-d252f38f71d0.jpg)
![6](https://user-images.githubusercontent.com/12046302/172040077-3ff713bb-d399-4f9b-809a-75a7004d140b.jpg)
![5](https://user-images.githubusercontent.com/12046302/172040079-616acf16-5aeb-4796-bc71-b4d00300d323.jpg)

**Comment by fmw42** [comment]
try

`convert document.jpg -negate -lat 20x20+10% -negate document_cleaned.jpg`

![document_cleaned](https://user-images.githubusercontent.com/13458218/172061295-d40e2852-2c06-45ee-8084-e4c32bfeaa7b.jpg)

Getting rid of the noise is harder. Change convert to magick if on Imagemagick 7

**Comment by snibgo** [comment]
Another approach is to aim for black text on a white background, recognising that most pixels should become white.
```
magick 172040057-acc717da-037b-4202-a4d8-d252f38f71d0.jpg -contrast-stretch 1,80% x.png
```
![x](https://user-images.githubusercontent.com/33812368/172063357-71e99209-7ca8-4936-9297-21194ca7c50a.png)

**Reply by srini8080** [reply]
Thanks, -lat worked.
